.cloudbuild/library_generation/image-configuration/airlock-pypirc

```diff
@@ -0,0 +1,9 @@
+[distutils]
+index-servers = ah-3p-staging-python
+# TODO: use this index instead when `lxml` and `versions` are available in the `trusted` airlock registry
+# index-servers = python-3p-trusted
```

Comment:
Could you point to which doc/webpage we should look at whether it's available?

Reply:
Yes, I added a link to the package lookup tool.